### PR TITLE
Objects revisited - aaron he

### PR DESCRIPTION
**Pull request by AaronHe09**
Screenshot 2023-06-19 103109

### CODE REVIEW

File: objects-revisited/main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+function deleteEmployee(employeeName) {
+  delete business.employees[employeeName];
+  business.totalEmployees--;
+}
+
+addEventListener('DOMContentLoaded', function () {
+  addWeekends();
+  addEmployees();
+  deleteEmployee('Raymond');
+});
```
Comment (shawnkost) [inline objects-revisited/main.js:75]: You also need to add the `fullTime` property to employees inside of here. Right now, you are currently only adding that property to employees that get created after initialization. The employees that exist before aren't getting this property. So "Raul" for example, currently does not have a `fullTime` property

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Aaron! Let me know if you have any questions

File: objects-revisited/main.js (hunk 1 above)
Comment (AaronHe09) [reply objects-revisited/main.js]: image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**